vhdbuilder/packer/install-dependencies.sh

```diff
@@ -97,10 +97,9 @@ cat << EOF >> ${VHD_LOGS_FILEPATH}
 EOF
 
 VNET_CNI_VERSIONS="
+1.2.0
 1.1.8
-1.1.7
```

Comment:
just to confirm 1.1.7 is not in use (or will not be in use by the time this VHD is released?)

Reply:
Yes we finished toggle release to 1.1.8 two weeks ago.